llvm/lib/Frontend/Offloading/OffloadWrapper.cpp

```diff
@@ -70,7 +70,7 @@ StructType *getBinDescTy(Module &M) {
 }
 
 PointerType *getBinDescPtrTy(Module &M) {
-  return PointerType::getUnqual(getBinDescTy(M));
+  return PointerType::getUnqual(M.getContext());
 }
```

Comment:
Both this and getDeviceImagePtrTy give the same type. Will look into if deduplicating makes sense.